stories/scss/action-icon/action-icon-basic.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { Meta, Story } from '@storybook/angular';
+
+interface ActionBasicStory {
+	mod: string;
+	palette: string;
+	state: string;
+	disabled: boolean;
+}
```

Comment:
Pour que le build passe :
```suggestion
   size: string;
}
```